### PR TITLE
Blog: Node-RED Manufacturing Architecture

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

Blog Article about a Node-RED Manufacturing Architecture based on the Purdue / ISA-99 Model

## Related Issue(s)

 - [ ] https://github.com/FlowFuse/website/issues/1823
 - [ ] https://github.com/FlowFuse/website/issues/1824

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by gdziuba** [comment]
@MarianRaphael Are you waiting on me for this?

File: src/blog/2024/03/node-red-architecture.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+**Enterprise Network:** Within the higher echelons of the network, FlowFuse establishes its domain, serving as a central hub for managing multiple Node-RED instances. In this space, Node-RED’s agility is leveraged for creating test environments, where simulated data flows can be used to model and understand potential changes before they are deployed on the shopfloor. For enterprise-wide KPI calculations, advanced analytics, and integration with other business systems like ERP or CRM. This layer can use data processed at the edge to inform enterprise-level decisions, drive continuous improvement, and harness machine learning models to glean deeper insights into the production process, quality control, and supply chain logistics.
```
Comment (gdziuba) [inline src/blog/2024/03/node-red-architecture.md:36]: Thoughts about including Citizen Development here?  Then link to a Citizen Development blog of your choice.  @Hasmin-AC would like the interlinking.  😉

File: src/blog/2024/03/node-red-architecture.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+
+### How to communicate between Node-RED Instance? Project Nodes
+
+Facilitating communication between Node-RED instances is a fundamental requirement for creating a cohesive and responsive Node-RED architecture. FlowFuse elevates this capability by providing an event-driven communication framework (based on MQTT) that binds different Node-RED instances together. This is primarily achieved through the use of Project Links—a feature within FlowFuse that allows for the smooth transfer of data between instances.
+
+These Project Links are more than just communication channels; they represent a method of organizing Node-RED instances into a networked application, where each instance can be considered a node within the project. With Project Links, instances can subscribe to specific events or topics and publish messages that other instances are listening for. This is particularly useful for triggering actions across the network, like updating a dashboard in real time or controlling devices on the shopfloor based on analytics computed at the edge.
```
Comment (gdziuba) [inline src/blog/2024/03/node-red-architecture.md:48]:
```suggestion
These [Project Links](https://flowfuse.com/docs/user/projectnodes/) are more than just communication channels; they represent a method of organizing Node-RED instances into a networked application, where each instance can be considered a node within the project. With Project Links, instances can subscribe to specific events or topics and publish messages that other instances are listening for. This is particularly useful for triggering actions across the network, like updating a dashboard in real time or controlling devices on the shopfloor based on analytics computed at the edge.
```

File: src/blog/2024/03/node-red-architecture.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+title: Node-RED Manufacturing Architecture
+subtitle: Enhancing Factory Operations with Node-RED and FlowFuse
+description: This article explores the integration of Node-RED and FlowFuse within a factory's multilayered infrastructure, highlighting the strategic organization of data and connectivity from the shopfloor to the enterprise level for improved operational efficiency.
+authors: ["marian-demme"]
+tags:
+    - posts
+    - node-red
+    - flowfuse
```
Comment (gdziuba) [inline src/blog/2024/03/node-red-architecture.md:9]:
```suggestion
    - flowfuse
    - manufacturing
    - perdue model
    - ISA 99
```

**Comment by gdziuba** [comment]
I made my suggestions above.  Let me know when it is ready for review, and I will approve.  I am excited to see content like this.

**Review by gdziuba — APPROVED** [review]

**Review by Hasmin-AC — APPROVED** [review]

**Pull request merged**